AutoCollection/CorrelationContextManager.ts

```diff
@@ -215,12 +216,15 @@ export class CorrelationContextManager {
 
         // We need this loop to copy outer methods like Error.captureStackTrace
         var props = Object.getOwnPropertyNames(orig);
-        for(var i=0; i<props.length; i++) {
+        for(var i=0; i < props.length; i++) {
             var propertyName = props[i];
             if (!AppInsightsAsyncCorrelatedErrorWrapper[propertyName]) {
                 Object.defineProperty(AppInsightsAsyncCorrelatedErrorWrapper, propertyName, Object.getOwnPropertyDescriptor(orig, propertyName));
             }
         }
-        global.Error = AppInsightsAsyncCorrelatedErrorWrapper;
+        
+        // explicit cast to <any> required to avoid type error for captureStackTrace
+        // with latest node.d.ts (despite workaround above)
+        global.Error = <any>AppInsightsAsyncCorrelatedErrorWrapper;
```

Comment:
Maybe we could avoid this with the error wrapper being a class that inherits from global.Error in explicit TS rather than the manual prototype assignment used here. However, I suspect that will cause some issues however. I'm okay with the cast to any